apps/gamification/pages/api/dashboard/all-quests-info.ts

```diff
@@ -28,40 +25,43 @@ const zQuery = z.object({
   pageSize: z.coerce.number().max(100).nullable(),
 })
 
-const handler = withSiweAuth(
-  withDashboardAllowlistAuth(async (req, res) => {
-    if (!process.env.GAMIFICATION_DASHBOARD_API || !req.query || req.method !== 'GET') {
-      return res.status(400).json({ message: 'API URL Empty / Method wrong' })
-    }
+const handler = async (req, res) => {
+  if (!process.env.GAMIFICATION_DASHBOARD_API || !req.query || req.method !== 'GET') {
+    return res.status(400).json({ message: 'API URL Empty / Method wrong' })
+  }
+
+  if (!req?.headers?.authorization) {
+    return res.status(400).json({ message: 'Header Authorization Empty' })
+  }
 
-    const queryString = qs.stringify(req.query, { arrayFormat: 'comma' })
-    const queryParsed = qs.parse(queryString)
-    const parsed = zQuery.safeParse(queryParsed)
+  const queryString = qs.stringify(req.query, { arrayFormat: 'comma' })
+  const queryParsed = qs.parse(queryString)
+  const parsed = zQuery.safeParse(queryParsed)
 
-    if (parsed.success === false) {
-      return res.status(400).json({ message: 'Invalid query', reason: parsed.error })
-    }
+  if (parsed.success === false) {
+    return res.status(400).json({ message: 'Invalid query', reason: parsed.error })
+  }
 
-    const response = await fetch(
-      `${process.env.GAMIFICATION_DASHBOARD_API}/quests/org/${queryParsed.address}?${queryString}`,
-      {
-        method: 'GET',
-        headers: {
-          Accept: 'application/json',
-          'Content-Type': 'application/json',
-          'x-secure-token': process.env.DASHBOARD_TOKEN as string,
-        },
+  const response = await fetch(
+    `${process.env.GAMIFICATION_DASHBOARD_API}/quests/org/${queryParsed.address}?${queryString}`,
+    {
+      method: 'GET',
+      headers: {
+        Authorization: req?.headers?.authorization as string,
+        Accept: 'application/json',
+        'x-secure-token': process.env.DASHBOARD_TOKEN as string,
```

Comment:
since BE dashboard api already has whitelist control, we don't need this secure token anymore and we can send the request from FE to BE directly